### ISSUE TITLE
Deck stats?

### ISSUE THREAD

**Issue by mtgred** [Kungsgeten/netrunner#5]
Is there a way to show the number of card or influence in a deck?
Showing the influence cost next to each card would be awesome.

**Comment by Kungsgeten** [comment]
No, not at the moment. I've been thinking about adding features like that,
but it would take quite a lot of work. Perhaps if I get inspired :) I would
be happy to welcome a feature like this, if you'd want to implement it.
